external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -235,4 +242,16 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationSuite {
     assert(row(0).getString(3) === "mnop")
     assert(row(0).getString(4) === "q")
   }
+
+  test("SPARK-32576: character array type tests") {
+    val df = sqlContext.read.jdbc(jdbcUrl, "char_array_types", new Properties)
+    val row = df.collect()
+    assert(row.length == 1)
```

Comment:
super nit: `===`

Reply:
oh, I missed that. But, its trivial, so I'll fix it next time I update this file. Anyway, thanks for the check, @yaooqinn

Reply:
That's alright. We don't currently have a style rule for that. `==` is already being used here and there too though I prefer `===` as well because it can compare, for example, arrays properly with a good error message.